api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -780,6 +796,16 @@ public void endVisit_shouldFailIfValidationErrorsAreFound() {
 		visitService.endVisit(visit, cal.getTime());
 	}
 	
+	@Test
+	public void endVist_shouldSetTheStopDateTimeOfVisit() {
+		Visit visit = visitService.getVisit(1);
+		
+		assertNull(visit.getStopDatetime());
```

Comment:
How are you confirming that the stop date and time of the visit is set?

Reply:
@dkayiwa  i have set the stop date and time